### PR TITLE
HTBHF-1793 Added autofocus to the last child name when there's more t…

### PR DESCRIPTION
**Pull request by dwybourn**
Added autofocus for child's name on child dob page. Only added for last entry and if there's more than one input.

Went for an approach different to that mentioned in the jira story. Happy to change it if we feel another way is better.

### CODE REVIEW

File: src/web/views/children-dob.njk [hunk 1]
```diff
@@ -25,7 +26,8 @@
         legend: aboutYourChild,
         dateLegend: dateOfBirth,
         removeButton: removeChild
-      }
+      },
+      autofocus: i == children.inputCount - 1 and children.inputCount > 1
```
Comment (gethyn1) [inline src/web/views/children-dob.njk]: This should use strict equality operator `===`

Comment (gethyn1) [inline src/web/views/children-dob.njk]: Does this introduce it's own a11y issues? When a user with more than one child chooses to change details or use the back button, they will always be focused on the last input.

Comment (dwybourn) [reply src/web/views/children-dob.njk]: Just thought of that and addressing that now.

**Comment by dwybourn** [comment]
> Happy for this to go in as is, but I'd like to do a bit more in another PR. Can we do something like adding a `req.session.children.autofocus=n` property in `handleAddAction`, and unset that property in `setChildrenInSessionForPost`? As `behaviourForPost` is defined as:
> 
> ```
> 
> const behaviourForPost = [
>   setChildrenInSessionForPost,
>   handleAddAction,
>   handleRemoveAction
> ]
> ```
> 
> I think that means that the property would be unset on every post, and reset if the action is to add a child.

@YetAnotherMatt Added as part of this PR.

File: src/web/views/children-dob.njk (hunk 1 above)
Comment (dwybourn) [reply src/web/views/children-dob.njk]: Updated to ===

File: src/web/views/macros/htbhf-child-dob-input.njk [hunk 2]
```diff
@@ -17,6 +17,7 @@
         }
      }) %}
 
+
```
Comment (gethyn1) [inline src/web/views/macros/htbhf-child-dob-input.njk]: Rogue new line

Comment (dwybourn) [reply src/web/views/macros/htbhf-child-dob-input.njk]: Appears to be a git error.

**Pull request merged**